CHANGELOG.md

```diff
@@ -92,6 +92,10 @@ Contributors:
 
 - Fix compiled sql for ephemeral models ([#3139](https://github.com/fishtown-analytics/dbt/pull/3139), [#3056](https://github.com/fishtown-analytics/dbt/pull/3056))
 
+### Fixes
+
+- Fix compiled sql for ephemeral models ([#3139](https://github.com/fishtown-analytics/dbt/pull/3139), [#3056](https://github.com/fishtown-analytics/dbt/pull/3056))
+
```

Comment:
Do you want to update the PR number and move up below `dbt 0.20.0 (Release TBD)`?